src/Files.App.Server/Files.App.Server.csproj

```diff
@@ -1,4 +1,5 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<!--  Copyright (c) 2018-2024 Files Community. Licensed under the MIT License.  -->
```

Comment:
For other reviewers: this is the reason why addition and deletion aren't the same count (newly the header added)